arbitrum-docs/launch-orbit-chain/how-tos/orbit-chain-finality.md

```diff
@@ -0,0 +1,29 @@
+# Orbit Chain Confirmation and Finality
+
+## Child Chain Transactions
```

Comment:
```suggestion
## Child chain transactions
```
Titles should be in sentence-case ([Style conventions](https://docs.arbitrum.io/for-devs/contribute#style-conventions))

Reply:
(Apply on all other content labels 🙏 )